.github/workflows/jacoco_report.yml

```diff
@@ -25,5 +26,6 @@ jobs:
         with:
           paths: ${{ github.workspace }}/airbyte-integrations/connectors/source-postgres/build/reports/jacoco/test/jacocoTestReport.xml
           token: ${{ secrets.GITHUB_TOKEN }}
-          min-coverage-overall: 40
-          min-coverage-changed-files: 60
+          min-coverage-overall: 64
+          title: Coverage report for source-postgres
```

Comment:
When we extend the coverage to other connectors, the title will need to be templatized so that it does not keep showing source-postgres. Don't think it is blocking the merge on this PR.

Reply:
It's not blocking. We can extend it by running the gradle test command for multiple db sources. We can use this [tool](https://docs.gradle.org/current/userguide/jacoco_report_aggregation_plugin.html) to display an aggregate report